graphics/sheepit-client/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               gitlab 1.0
+PortGroup               java 1.0
+PortGroup               app 1.0
+
+name                    sheepit-client
+gitlab.setup            sheepitrenderfarm client 6.22198.0 v
+revision                0
+
+categories              graphics java
+license                 GPL-2
+maintainers             {judaew @judaew} openmaintainer
+
+description             SheepIt Render Farm Client is an Open Source client for \
+                        the distributed render farm SheepIt.
+long_description        {*}${description}
+homepage                https://www.sheepit-renderfarm.com
+
+checksums               rmd160  f4643b763db8a1d53043ce099c34408b80270955 \
+                        sha256  19167854c85b5f901794eac95b57ef62ad189999644194ced6c90080f7c129d5 \
+                        size    123769982
+
+java.version            11+
+java.fallback           openjdk17
+use_configure           no
+
+depends_build-append    port:gradle port:makeicns
+set target_dir          ${prefix}/share/java/sheepitrenderfarm
+
+build.env-append        GRADLE_USER_HOME=${worksrcpath}
+build.cmd               ${prefix}/bin/gradle
+build.target            shadowJar
+
+post-build {
+    system -W ${worksrcpath}/dmg-wrapper \
+        "${prefix}/bin/makeicns -in SheepIt.png -out SheepIt.icns"
```

Comment:
Do you need the direct call to `makeicns`? If you specify the png file to `app.icon`, this will be done automatically. (And the dep for `makeicns` will be automatically added via the pg.)

Reply:
Oh, you're right. I missed these lines when looking through app.tcl:

https://github.com/judaew/macports-ports/blob/d86d4e0a6087f68f25f6b88bc6154a76d7714736/_resources/port1.0/group/app-1.0.tcl#L286-L291

Done